### ISSUE TITLE
Hyper Parameter

### ISSUE THREAD

**Issue by ZHANGDANA607** [felixshing/FedAWS_Reproduce#1]
Thank you for your significant contribution to FL. I would like to ask whether, besides changing the model name to 'fedaws', there are other hyperparameters or model settings that need to be modified after downloading the code from GitHub? During the training process, the prediction accuracy on the test set has consistently remained between 11-14%, which has greatly troubled me.
